data/domain/cards.tsx

```diff
@@ -148,6 +164,29 @@ export const updateCards = (data: Map<string, any>) => {
     const players = data.get("players") as Player[];
     const optLacc = data.get("OptLacc");
 
+    if (rift.bonuses.find(bonus => bonus.name == "Skill Mastery")?.active) {
+
+        const skillMastery = rift.bonuses.find(bonus => bonus.name == "Skill Mastery") as SkillMastery;
```

Comment:
minor: this can be moved above the `if` and then change the if to simply do `skillMastery.active`. Just more efficient then doing a double `find`

Reply:
Indeed good catch